content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: blog
+title: "Introducing ClusterClass and managed topologies in Cluster API"
```

Comment:
Could you please convert this to title case for consistency, that is, `Introducing ClusterClass and Managed Topologies in Cluster API` 😄